OpenSim/Common/Property.h

```diff
@@ -876,6 +882,14 @@ class SimpleProperty : public Property<T> {
         SimTK::writeUnformatted(o, values);
     }
 
+    // This is the default implementation. Specializations for types that
+    // contain floats (e.g., double, Vec3, Vec6, Vector, Transform) are
+    // provided below to control the number of digits shown.
+    void writeSimplePropertyToStreamForDisplay(std::ostream& o, 
+                                               const int precision) const {
+        writeSimplePropertyToStream(o);
```

Comment:
Similar to above: the comment should say that by default, the precision is ignored.